bigbluebutton-html5/imports/ui/components/whiteboard/service.js

```diff
@@ -32,19 +32,23 @@ function handleAddedAnnotation({
   }
 
   const fakeAnnotation = Annotations.findOne({ id: `${annotation.id}-fake` });
-  const fakePoints = fakeAnnotation.annotationInfo.points;
-  const { points: lastPoints } = annotation.annotationInfo;
-
-  if (annotation.annotationType !== 'pencil') {
-    Annotations.update(fakeAnnotation._id, {
-      $set: {
-        position: annotation.position,
-        'annotationInfo.color': isEqual(fakePoints, lastPoints) || annotation.status === DRAW_END ?
-          annotation.annotationInfo.color : fakeAnnotation.annotationInfo.color,
-      },
-      $inc: { version: 1 }, // TODO: Remove all this version stuff
-    });
-    return;
+  let fakePoints;
+
+  if (fakeAnnotation) {
```

Comment:
Is this related to this PR?

Reply:
Yes, If I remember correctly is preventing an error from 'annotationInfo' of undefined when the poll results are published.